<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportLigParGenDialog</class>
 <widget class="QDialog" name="ImportLigParGenDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>385</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Import Forcefield Wizard</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QWidget" name="WizardHeaderWidget" native="true"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="MainStack">
     <property name="minimumSize">
      <size>
       <width>319</width>
       <height>120</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="SelectFilePage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>OpenMM File</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>4</number>
           </property>
           <item>
            <widget class="QLineEdit" name="InputXMLEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="InputXMLButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="main.qrc">
               <normaloff>:/menu/icons/menu_open.svg</normaloff>:/menu/icons/menu_open.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>XYZ File</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>4</number>
           </property>
           <item>
            <widget class="QLineEdit" name="InputXYZEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="InputXYZButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="main.qrc">
               <normaloff>:/menu/icons/menu_open.svg</normaloff>:/menu/icons/menu_open.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="PreviewSpeciesPage">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QFrame" name="MainViewFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>300</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="SpeciesViewer" name="SpeciesView" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>3</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>CrossCursor</cursorShape>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="focusPolicy">
             <enum>Qt::WheelFocus</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="PreviewTermsPage">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTreeView" name="xmlTree"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="AtomTypesPage">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QListView" name="AtomTypeList"/>
       </item>
       <item>
        <widget class="QGroupBox" name="SimplifyAtomTypesGroup">
         <property name="title">
          <string>SImplify Atom Types</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Collapse atom types with the same short-range interaction parameters into single types. Charges are assigned to species atoms.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="ReduceToMasterTermsGroup">
         <property name="title">
          <string>Reduce to Master Terms</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8" stretch="0">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Create master terms for intramolecular interactions, grouping those with identical functional forms and parameters together.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ApplyForcefieldPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>113</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="CreateForcefieldGroup">
         <property name="title">
          <string>Create Forcefield</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="horizontalSpacing">
           <number>4</number>
          </property>
          <property name="verticalSpacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="label_name">
            <property name="text">
             <string>Forcefield Name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="ForcefieldNameEdit"/>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>This will create a forcefield entry for the current parameters so that they can be re-applied to additional species</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>112</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="WizardFooterWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpeciesViewer</class>
   <extends>QWidget</extends>
   <header>gui/speciesViewer.hui</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>InputXMLEdit</tabstop>
  <tabstop>InputXMLButton</tabstop>
 </tabstops>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>nextStack()</slot>
  <slot>prevStack()</slot>
  <slot>xmlString(QString)</slot>
  <slot>nameString(QString)</slot>
  <slot>xmlFileDialog()</slot>
 </slots>
</ui>
